addons/website/static/src/snippets/s_instagram_page/000.scss

```diff
@@ -0,0 +1,5 @@
+.s_instagram_page {
+    iframe {
+        width: 100%;
+    }
+}
```

Comment:
I do this because I know we don't like inline css but here it might be worth putting this little rule inline in order to have one less asset ? :thinking: